test/node/msw-api/req/passthrough.node.test.ts

```diff
@@ -88,3 +97,35 @@ it('performs a request as-is if nothing was returned from the resolver', async (
     name: 'John',
   })
 })
+
+it.each([{ code: 204 }, { code: 205 }, { code: 304 }])(
+  'performs a $code request as-is if nothing was returned from the resolver',
```

Comment:
`$code` looks like a non-resolved template string value, does it not?

Reply:
`.each` extracts that `$code` and inlines the value from the object. 

Playwright doesn't support it but vitest does.  we could do either way, but I was just doing this style here because the runner supports it.